projects/element-ng/autocomplete/si-autocomplete-listbox.directive.ts

```diff
@@ -115,6 +108,9 @@ export class SiAutocompleteListboxDirective<T> implements OnInit, OnChanges, Aft
   }
 
   get active(): SiAutocompleteOptionDirective<T> | null {
-    return this.keyManager?.activeItem ?? null;
+    if (this.keyManager.activeItemIndex === null) {
+      return null;
+    }
+    return this.options().at(this.keyManager.activeItemIndex) ?? null;
```

Comment:
Why this change? 

`activeItem` should still be working

Reply:
when options get updated dynamically the `keyManager.activeItem` still refs to the old option item. could be bug in cdk as activeItemIndex is updated correctly.